docs/guides/stETH & wstETH Integration Paper.md

```diff
@@ -0,0 +1,396 @@
+# stETH & wstETH Integration Paper
+
+[![Supported by LEGO](https://img.shields.io/badge/Supported%20by-LEGO-%2300A3FF)](https://www.notion.so/LEGO-Lido-Ecosystem-Grants-Organisation-d7f0bf0182d44348b6173639d2e8363d)  
+
+
+### What stETH is
+
+Normally users need to run own node and require at least 32ETH to participate in ETH 2.0. Funds will be locked until it is released. But with LIDO which allows users to stake their ETH into the ETH2.0 PoS network, anyone can stake  ETH and get back stETH + the x% staking reward from ETH2.0. The benefit of this is that stETH can be swapped for ETH at any time so users are no longer locked until ETH2.0 is released. The stETH token is a tokenized version of staked ether. 
+
+When a user sends ether into the Lido liquid staking smart contract, the user receives the corresponding amount of stETH tokens. The stETH token represents Lido user’s deposits and the corresponding staking rewards and slashing penalties. It is also is a liquid alternative for the staked ether: it could be transferred, traded, or used in DeFi applications. The stETH token balance is be calculated based on the total amount of staked ether, plus rewards and minus any slashing penalties.
+
+#
+
+<img width="1125" src="https://github.com/OffcierCia/stETH-wstETH-Integration-paper/blob/main/01.png?raw=true" />
+
+
+Lido makes the stETH token balance track a balance of corresponding balance of beacon chain ether. A user’s balance of stETH tokens corresponds 1 to 1 to an amount of ether a user could receive if withdrawals were enabled and instant.
+
+The DAO selects node operators, which also validate transactions on the beacon chain and adds their addresses to the `NodeOperatorsRegistry` contract. Authorized node operators have to generate a set of keys for the validation and also provide them with the smart contract. As ether is received from users, it is distributed in chunks of 32 Ether between all active node operators. The staking pool contract contains a list of node operators, their keys, and the logic for distributing rewards between them.
```

Comment:
Can't tell which contract is called "the staking pool contract" here, but the Node Operators list with the keys is in [NodeOperatorsRegistry](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol) contract, and the rewards distribution is in [Lido](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/Lido.sol)

Reply:
+ Fixed